### DISCUSSION TITLE
The labels of points in Pole Figure

### DISCUSSION THREAD

**Discussion by louis96108** [mtex-toolbox/mtex#2082] [Ask Anything] [answered]
![极图示意图](https://github.com/mtex-toolbox/mtex/assets/69557894/f137982d-6b89-4e66-8c81-3f9f9abb8e61)
I've printed the Pole figure by the Euler angles. How to add the label shown in the picture in the Pole figure？
Thanks a lot

**Comment by kilir** [comment, marked as answer]
Hi,
`plotPDF` takes the argument `'label'` followed by ay list of `{labels}`.
Cheers,
Rüdiger

**Reply by louis96108** [reply]
Hi Rüdiger，
For an orientation, its projection on the Pole Figure will be several points, for example, on the (111) Pole figure, the points will represent (111), (-111),(1-11)...,   So How to get MTEX to automatically label the crystal plane represented by each point. Thanks a lot.
Sincerely，
Louis

**Reply by kilir** [reply]
Hi Louis,

for a projection, the point you are plotting is also not  e.g. (1,1,1) but a vector which is parallel to (1,1,1) of a crystal with a given orientation, not a crystal plane.
However, you can specify a list of labels or generate using `Miller/char`. Simply symmetrise your Millers before computing the directiosn in the polefigure and generate the label accordingly.


```Matlab

cs = crystalSymmetry('m-3m');
ori =orientation.rand(1,cs);
h = symmetrise(Miller(1,1,0,cs));
plot(ori.* h,'label',char(h,'cell'))

```

Cheers,

Rüdiger

**Reply by JulianMeck** [reply]
Can you change the positions of the labels. I get some overlapping when they are all below the point.
Thanks
Julian

**Reply by ralfHielscher** [reply]
You can use the option `'textAboveMarker'` to position the label above the marker.
Ralf.

**Comment by Lu-Peter** [comment]
This code could be better?
plot(ori*symmetrise(h),'label',char(**symmetrise(h)**,'cell'),'upper');
